### PR TITLE
chore(EMS-3095): No PDF - Various DB/keystone field additions, improvements

### PR DESCRIPTION
**Pull request by ttbarnes**
## Introduction :pencil2:
This PR updates the keystone schema and DB fields in order to:
- Progress the upcoming section of a no PDF application ("export contract")
- Align/simplify some financial detail fields.

## Resolution :heavy_check_mark:
- Update DB dump.
- Create new `createAPrivateMarket` helper function.
- Exract export contract table/relationship creation into its own helper function, `createAnExportContract`.
- Rename financial details fields so that:
  - Salt/hash is not mentioned, instead, we have "vector".
  - Bank address is a simple text string, since this is publicly available information and therefore is not sensitive.
- Create new `PrivateMarket` table with an `ExportContract` relationship.
- Update and add avrious missing var chars for "name" fields.
- Update `createAnApplication` GQL mutation and m`createFullApplication` test helper function to create an export contract relationship.

## Miscellaneous :heavy_plus_sign:
- Fix some typos.

### CODE REVIEW

File: src/api/helpers/create-an-export-contract/index.test.ts [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+    expect(exportContract.finalDestinationKnown).toEqual(APPLICATION.DEFAULT_FINAL_DESTINATION_KNOWN);
+    expect(exportContract.goodsOrServicesDescription).toEqual('');
+    expect(exportContract.paymentTermsDescription).toEqual('');
+  });
+
+  test('it should return empty privateMarket fields', async () => {
+    const { privateMarket } = await createAnExportContract(context, application.id);
+
+    expect(privateMarket.attempted).toEqual(null);
```
Comment (Zainzzkk) [inline src/api/helpers/create-an-export-contract/index.test.ts:48]: not a blocker.  Can use `.toBeNull()` here

File: src/api/package.json [hunk 2]
```diff
@@ -19,7 +19,7 @@
     "lint": "eslint ./",
     "lint:fix": "eslint ./ --fix",
     "start": "keystone start --no-ui",
-    "test": "jest --runInBand --verbose --silent --coverage --config=jest.config.js"
```
Comment (Zainzzkk) [inline src/api/package.json:22]: should this be reverted back?

Comment (ttbarnes) [reply src/api/package.json:22]: Whoops, yes

File: src/api/helpers/create-an-export-contract/index.test.ts (hunk 1 above)
Comment (ttbarnes) [reply src/api/helpers/create-an-export-contract/index.test.ts:48]: Good spot, i've updated all instances of this in the API

**Review by Zainzzkk — APPROVED** [review]

File: src/api/package.json [hunk 3]
```diff
@@ -22,10 +22,10 @@
     "test": "jest --runInBand --verbose --silent --coverage --config=jest.config.js"
   },
   "dependencies": {
-    "@babel/core": "^7.24.3",
-    "@babel/plugin-transform-runtime": "^7.24.3",
-    "@babel/preset-env": "^7.24.3",
-    "@babel/preset-typescript": "^7.24.1",
+    "@babel/core": "^7.24.1",
```
Comment (Zainzzkk) [inline src/api/package.json:25]: I think you accidentally rolled back some packages

**Review by Zainzzkk — APPROVED** [review]

**Pull request merged**